src/components/profile/profile.stories.js

```diff
@@ -1,24 +1,44 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { text, boolean } from '@storybook/addon-knobs';
-import Profile, { OriginalProfile } from './profile';
+import { text, boolean, select } from '@storybook/addon-knobs';
+import { ThemeProvider } from 'styled-components';
+import Profile, { OriginalProfile } from './profile.component';
 import { info, notes } from './documentation';
+import classicTheme from '../../style/themes/classic';
+import OptionsHelper from '../../utils/helpers/options-helper';
 
 storiesOf('Profile', module)
   .add('default', () => {
     const email = text('email', 'johnsmith@sage.com');
     const initials = text('initials', 'JS');
-    const large = boolean('large', OriginalProfile.defaultProps.large);
+    const size = select('size', OptionsHelper.sizesPortrait, OptionsHelper.sizesPortrait[0]);
     const name = text('name', 'John Smith');
 
     return (
       <Profile
         email={ email }
         initials={ initials }
-        large={ large }
+        size={ size }
         name={ name }
       />
     );
+  })
+  .add('classic', () => {
+    const email = text('email', 'johnsmith@sage.com');
+    const initials = text('initials', 'JS');
+    const large = boolean('large', OriginalProfile.defaultProps.large);
+    const name = text('name', 'John Smith');
+
+    return (
+      <ThemeProvider theme={ classicTheme }>
+        <Profile
+          email={ email }
+          initials={ initials }
+          large={ large }
```

Comment:
Is this still correct? Should it use `size` now?

Reply:
It's only for the classic